src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -64,6 +64,12 @@ internal class ConnectionManager : IConnectionManager {
             _userConnections = new ConcurrentDictionary<Uri, IConnection>(userConnections);
 
             UpdateRecentConnections();
+
+            _shell.Idle += OnIdle;
+        }
+
+        private void OnIdle(object sender, EventArgs e) {
```

Comment:
Ok, then I'll merge your changes and add an idle verification inside VsStatusBar itself

---
In reply to: [78588775](https://github.com/Microsoft/RTVS/pull/2296#discussion_r78588775) [](ancestors = 78588775)